pointcloud_clustering/positionRPY position
bool is_Known